### PR TITLE
Tophat update

### PR DESCRIPTION
**Pull request by nekrut**
Updated dependencies and help for individual parameters. Now ready for the latest TopHat2 2.0.14 and Bowtie 2.2.5

### CODE REVIEW

**Comment by nekrut** [comment]
@dannon @blankenberg @jmchilton @erasche @bgruening @nsoranzo - please take a look. I would like to put it on test asap

File: tools/tophat2/tool_dependencies.xml [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-      <repository name="package_bowtie2_2_1_0" owner="devteam" prior_installation_required="False" />
+  <package name="bowtie2" version="2.2.5">
+      <repository name="package_bowtie_2_2_5" owner="iuc" prior_installation_required="False" />
     </package>
-    <package name="samtools" version="0.1.18">
-      <repository name="package_samtools_0_1_18" owner="devteam" prior_installation_required="False" />
-    </package>
-    <package name="tophat2" version="2.0.9">
-      <repository name="package_tophat2_2_0_9" owner="devteam" prior_installation_required="False" />
+    <package name="tophat" version="2.0.14">
```
Comment (bgruening) [inline tools/tophat2/tool_dependencies.xml]: It seems that the new package in the IUC repos is called `tophat2`: https://github.com/galaxyproject/tools-iuc/pull/136/files

Comment (davebx) [inline tools/tophat2/tool_dependencies.xml]: That was unintentional, I'm changing that to tophat now.

Comment (nekrut) [inline tools/tophat2/tool_dependencies.xml]: @davebx will be changing this

File: tools/tophat2/tophat2_wrapper.xml [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
+<tool id="tophat2" name="Tophat" version="0.9">
     <!-- Wrapper compatible with Tophat version 2.0.0+ -->
     <description>Gapped-read mapper for RNA-seq data</description>
     <version_command>tophat2 --version</version_command>
     <requirements>
-        <requirement type="package" version="0.1.18">samtools</requirement>
-        <requirement type="package" version="2.1.0">bowtie2</requirement>
-        <requirement type="package" version="2.0.9">tophat2</requirement>
+        <requirement type="package" version="2.2.5">bowtie2</requirement>
+        <requirement type="package" version="2.0.14">tophat2</requirement>
```
Comment (bgruening) [inline tools/tophat2/tophat2_wrapper.xml]: @davebx has changed it to `tophat` so this needs to be changed here.

**Comment by bgruening** [comment]
I like documentation improvements: :+1:

File: tools/tophat2/tophat2_wrapper.xml (hunk 2 above)
Comment (nekrut) [inline tools/tophat2/tophat2_wrapper.xml]: @davebx will be changing it back ;)

**Comment by davebx** [comment]
Looks good :+1:

**Pull request merged**